packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -1255,4 +1255,60 @@ void main() {
     expect(find.text('C'), findsOneWidget);
     expect(find.text('D'), findsNothing);
   });
+
+  testWidgets('Correct custom expanded padding', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      MaterialApp(
+        home: SingleChildScrollView(
+          child: ExpansionPanelList(
+            expansionCallback: (int _index, bool _isExpanded) {},
+            children: <ExpansionPanel>[
+              ExpansionPanel(
+                headerBuilder: (BuildContext context, bool isExpanded) {
+                  return Text(isExpanded ? 'B' : 'A');
+                },
+                body: const SizedBox(height: 100.0),
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+
+    expect(find.text('A'), findsOneWidget);
+    expect(find.text('B'), findsNothing);
+    RenderBox box = tester.renderObject(find.byType(ExpansionPanelList));
+    final double oldHeight = box.size.height;
+    expect(box.size.height, equals(oldHeight));
+
+    // Now, expand the child panel.
+    await tester.pumpWidget(
```

Comment:
This call rebuilds the the entire widget tree altogether. Also, `expandedPadding` was not defined in the previous call. I would just build the entire tree once, and then call a tap on a button that updates the state of your app to expand the body of the `ExpansionPanel`.

Reply:
Good points, was caused by stupid copy/pasting existing tests.

As `setState()` is needed for this approach I tried reusing/extending the existing `SimpleExpansionPanelListTestWidget` class.
I hope this was the best option here, I'm open for your feedback.